onnx/onnx.proto

```diff
@@ -424,6 +425,7 @@ message TypeProto {
     // This field MUST be present for this version of the IR.
     optional int32 elem_type = 1;
     optional TensorShapeProto shape = 2;
+    extensions 100 to max;
```

Comment:
Do you want this for TypeProto or Tensor? The title of the PR says TypeProto, but it is being added to Tensor.

Reply:
Oh you are right, it should be `TypeProto.Tensor`